src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Browser SPA monitoring 2.0
+tags:
+  - Browser
+  - Single page app monitoring
+translate:
+  - jp
+metaDescription: The new browser SPA monitoring capability is a streamlined way to monitor your users' experience and app performance.
+freshnessValidatedDate: 2024-03-18
+---
+
+<Callout variant="important">
+    This release is in limited preview and only available to select customers. Going forward, we plan on gradually phasing out the existing browser SPA experience.
+    </Callout> 
+
+For customers monitoring browser single page applications (SPA), we're excited to announce an overhaul of our SPA monitoring functionality, meant to address numerous customer pain points:
+
+* Unusable latest versions: Frequent conflicts with third-party libraries and unreliable interaction capture plagued the existing agent, often rendering the latest version unusable.
+* Patchwork solutions: Addressing one issue with SPA patches often introduced another, creating a frustrating cycle of fixes and regressions.
+* Third-party library conflicts: Global wrapping, particularly around `Promises`, often disrupted code functionality due to conflicts with other libraries.
+* Performance bottlenecks: Conflicts with code using timers, RAF, and promise chaining led to performance issues, ranging from slowdowns to freezes.
+
+The Pro+SPAv2 agent is designed to eliminate these problems and provide a significantly improved monitoring experience. Key changes include:
```

Comment:
```suggestion
The upgraded SPA monitoring experience is designed to eliminate these problems and provide a significantly improved monitoring experience. Key changes include:
```

Reply:
"redesigned" or "simplified" may be a more accurate but still positive term